### PR TITLE
Bump versions for tiup related content for 5.3.0

### PR DESCRIPTION
**Pull request by qqqdan**
change content for 5.3.0

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: upgrade-tidb-using-tiup.md [hunk 1]
```diff
@@ -7,23 +7,24 @@ aliases: ['/docs-cn/dev/upgrade-tidb-using-tiup/','/docs-cn/dev/how-to/upgrade/u
 
 本文档适用于以下升级路径：
 
-- 使用 TiUP 从 TiDB 4.0 版本升级至 TiDB 5.2 及后续修订版本。
-- 使用 TiUP 从 TiDB 5.0 版本升级至 TiDB 5.2 及后续修订版本。
-- 使用 TiUP 从 TiDB 5.1 版本升级至 TiDB 5.2 及后续修订版本。
+- 使用 TiUP 从 TiDB 4.0 版本升级至 TiDB 5.3 及后续修订版本。
+- 使用 TiUP 从 TiDB 5.0 版本升级至 TiDB 5.3 及后续修订版本。
+- 使用 TiUP 从 TiDB 5.1 版本升级至 TiDB 5.3 及后续修订版本。
+- 使用 TiUP 从 TiDB 5.2 版本升级至 TiDB 5.3 及后续修订版本。
```
Comment (TomShawn) [inline upgrade-tidb-using-tiup.md:13]: @seiya-annie PTAL

**Review by shichun-0415 — APPROVED** [review]
LGTM

File: upgrade-tidb-using-tiup.md (hunk 1 above)
Comment (seiya-annie) [reply upgrade-tidb-using-tiup.md:13]: yes, according to test, tiup supported to upgrade from 4.0,5.0,5.1,5.2 to 5.3

**Review by seiya-annie — APPROVED** [review]

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**